gdk/gdk.rectangle.lisp

```diff
@@ -266,4 +267,16 @@
 
 (export 'gdk-rectangle-union)
 
+;;; ----------------------------------------------------------------------------
+;;; gdk_rectangle_equal ()
+;;; ----------------------------------------------------------------------------
+
+#+gdk-3-20
+(defcfun gdk-rectangle-equal :boolean
```

Comment:
I think this should be a Lisp-level function since it's being translated into a Lisp structure already?

Reply:
Ok, will do.